warehouse/query-core/src/main/java/datawave/query/predicate/EventDataQueryFieldFilter.java

```diff
@@ -37,7 +50,7 @@ public EventDataQueryFieldFilter(EventDataQueryFieldFilter other) {
      * @param script
      *            a script
      * @param nonEventFields
-     *            a set of non event fields
+     *            a set of non-event fields
      */
     public EventDataQueryFieldFilter(ASTJexlScript script, Set<String> nonEventFields) {
```

Comment:
This constructor can be deprecated in favor of `EventDataQueryFieldFilter(KeyProjection)`

Reply:
sure thing, marking as deprecated.